azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/SetupControlBarView.kt

```diff
@@ -92,128 +82,125 @@ internal class SetupControlBarView : LinearLayout {
-                    0,
-                    0
-                )
-                setupAudioDeviceButton.text = context.getString(
-                    R.string.azure_communication_ui_setup_audio_device_android
-                )
+                context.getString(R.string.azure_communication_ui_setup_audio_device_android)
             }
             AudioDeviceSelectionStatus.BLUETOOTH_SCO_SELECTED -> {
-                setupAudioDeviceButton.setCompoundDrawablesRelativeWithIntrinsicBounds(
-                    0,
-                    R.drawable.azure_communication_ui_ic_fluent_speaker_bluetooth_24_regular,
-                    0,
-                    0
-                )
-                setupAudioDeviceButton.text =
-                    context.getString(R.string.azure_communication_ui_setup_audio_device_bluetooth)
-            }
-            else -> {
-                setupAudioDeviceButton.text = ""
+                context.getString(R.string.azure_communication_ui_setup_audio_device_bluetooth)
             }
+            else -> { "" }
         }
 
-        val setupAudioDeviceButtonColor =
-            if (setupCameraButton.isSelected) R.color.azure_communication_ui_color_on_surface_camera_active
-            else R.color.azure_communication_ui_toggle_selector
-        setButtonColor(setupAudioDeviceButton, setupAudioDeviceButtonColor)
+        audioDeviceButton.isSpeakerON = audioDeviceSelectionStatus == AudioDeviceSelectionStatus.SPEAKER_SELECTED
+        audioDeviceButton.isReceiverON = audioDeviceSelectionStatus == AudioDeviceSelectionStatus.RECEIVER_SELECTED
+        audioDeviceButton.isBluetoothON = audioDeviceSelectionStatus == AudioDeviceSelectionStatus.BLUETOOTH_SCO_SELECTED
+        audioDeviceButton.refreshDrawableState()
     }
 
     private fun toggleAudio() {
-        if (setupAudioButton.isSelected) {
+        if (micButton.isSelected) {
             viewModel.turnMicOff()
         } else {
             viewModel.turnMicOn()
         }
     }
 
     private fun toggleVideo() {
-        if (setupCameraButton.isSelected) {
+        if (cameraButton.isSelected) {
             viewModel.turnCameraOff()
         } else {
             viewModel.turnCameraOn()
         }
     }
+}
 
-    private fun setButtonColor(button: Button, colorId: Int) {
-        button.setTextColor(ContextCompat.getColor(context, colorId))
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
-            button.compoundDrawableTintList =
-                ColorStateList.valueOf(ContextCompat.getColor(context, colorId))
-        } else {
-            button.compoundDrawables[1].colorFilter = PorterDuffColorFilter(
-                ContextCompat.getColor(context, colorId),
-                PorterDuff.Mode.SRC_IN
-            )
+internal class SetupButton(context: Context, attrs: AttributeSet?) :
+    androidx.appcompat.widget.AppCompatButton(context, attrs) {
```

Comment:
can we add `import androidx.appcompat.widget.AppCompatButton` to top of this file so then there is no need to type the full namespace here

Reply:
Done.